manifests/init.pp

```diff
@@ -101,8 +101,8 @@
   $apt_source              = 'https://apt.puppet.com',
   $mac_source              = 'https://downloads.puppet.com',
   $windows_source          = 'https://downloads.puppet.com',
-  $solaris_source          = 'puppet:///pe_packages',
-  $aix_source              = 'puppet:///pe_packages',
+  $solaris_source          = 'puppet:///modules/pe_packages',
+  $aix_source              = 'puppet:///modules/pe_packages',
```

Comment:
This is the path from where files are served by the server node and it will break functionality